web/rest/brand/src/Controller/Provider/Administrator/MassUpdateAclsAction.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Controller\Provider\Administrator;
+
+use Ivoz\Provider\Domain\Model\Administrator\AdministratorInterface;
+use Ivoz\Provider\Domain\Model\Administrator\AdministratorRepository;
+use Ivoz\Provider\Domain\Model\Company\CompanyRepository;
+use Service\AdministratorRelPublicEntity\SwitchAcl;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class MassUpdateAclsAction
+{
+    public function __construct(
+        private AdministratorRepository $administratorRepository,
+        private CompanyRepository $companyRepository,
+        private SwitchAcl $switchAcl
+    ) {
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        /** @var int[] $publicEntitiesRelUserIds */
+        $publicEntitiesRelUserIds = $request->toArray();
+        $adminId = (int)$request->attributes->get('id');
+        $routeName = (string)$request->attributes->get('_route');
+
+        /** @var ?AdministratorInterface $administrator */
+        $administrator = $this->administratorRepository->find($adminId);
+
+        if (is_null($administrator)) {
+            return new Response('Administrator not found', 404);
+        }
+
+        if (!$administrator->getRestricted()) {
+            return new Response('Administrator user is not restricted', 403);
+        }
+
+        if (is_null($administrator->getCompany())) {
+            return new Response('Does not exist a company associated to admin user', 403);
+        }
+
+        match (true) {
+            str_contains(
+                $routeName,
+                SwitchAcl::ACL_GRANT_ALL
+            ) => $this
+                ->switchAcl
+                ->execute(
+                    $adminId,
+                    $publicEntitiesRelUserIds,
+                    SwitchAcl::ACL_GRANT_ALL
+                ),
+            str_contains(
+                $routeName,
+                SwitchAcl::ACL_GRANT_READ_ONLY
+            ) => $this
+                ->switchAcl
+                ->execute(
+                    $adminId,
+                    $publicEntitiesRelUserIds,
+                    SwitchAcl::ACL_GRANT_READ_ONLY
+                ),
+            str_contains(
+                $routeName,
+                SwitchAcl::ACL_REVOKE_ALL
+            ) => $this
+                ->switchAcl
+                ->execute(
+                    $adminId,
+                    $publicEntitiesRelUserIds,
+                    SwitchAcl::ACL_REVOKE_ALL
+                ),
+            default => throw new \UnhandledMatchError('No matching route found.'),
+        };
+
+        return new Response('', 200);
```

Comment:
API is expecting a json response (+ set headers)
`        produces:
          - 'application/json'`